package.json

```
@@ -1,4 +1,10 @@
 {
+  "scripts": {
+    "fmt": "eslint . --fix",
+    "lint": "tsc --noEmit && eslint .",
+    "deploy:commands": "ts-node deploy-commands.ts",
+    "server": "ts-node index.ts"
```

```suggestion
    "dev": "ts-node index.ts"
```

`dev` is more appropriate/common imo!

---

`server` would be more appropriate when used to 'serve' the build. (We should probably add a script for that!)

---

Makes sense to me!